src/develop/masks/circle.c

```diff
@@ -864,7 +867,9 @@ static int dt_circle_get_area(dt_iop_module_t *module, dt_dev_pixelpipe_iop_t *p
   return 1;
 }
 
-static int dt_circle_get_mask(dt_iop_module_t *module, dt_dev_pixelpipe_iop_t *piece, dt_masks_form_t *form,
+/*static*/ int dt_circle_get_mask(const dt_iop_module_t *const restrict module,
```

Comment:
can you comment about why static is in comment ? If not needed we should just remove it anyway.

Reply:
My bad, I had commented it out so that the compiler would generate a non-inlined version for me to check the generated code, and simply forgot to remove the commenting.